.travis.yml

```diff
@@ -49,15 +45,3 @@ matrix:
       dist: trusty
```

Comment:
Does not need to be trusty in particular. We just need a Java 8 build

Reply:
Ack. Will move from oraclejdk8 to openjdk8 though.